ast/dml.go

```diff
@@ -791,6 +791,8 @@ type DeleteStmt struct {
 	Quick        bool
 	IsMultiTable bool
 	BeforeFrom   bool
+	// TableHints represents the level Optimizer Hint for join type.
```

Comment:
Do you mean "the table level optimizer hint"?

Reply:
Yeah. I just referred the previous comments.

Reply:
How about fixing these wrong comments in this PR?

Reply:
OK